projects/jackson-core/build.sh

```diff
@@ -15,6 +15,9 @@
 #
 ################################################################################
 
+# Copy seed corpus and dictionary.
```

Comment:
nit: again, would probably change comment to "move" instead of "copy to be accurate.